src/promptflow/promptflow/_sdk/_service/apis/trace.py

```diff
@@ -2,29 +2,63 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 
-from flask import jsonify
+from flask_restx import fields
 
 from promptflow._sdk._service import Namespace, Resource
 from promptflow._sdk._service.utils.utils import get_client_from_request
 
 api = Namespace("traces", description="Trace Management")
 
-
-@api.route("/<string:span_id>")
-class Trace(Resource):
-    @api.response(code=200, description="Get trace info")
-    @api.doc(description="Get trace")
-    def get(self, span_id: str):
-        trace = get_client_from_request()._traces.get(span_id=span_id)
-        print(trace)
-        return "Trace", 200
+context_model = api.model(
+    "Context",
+    {
+        "trace_id": fields.String(required=True),
+        "span_id": fields.String(required=True),
+        "trace_state": fields.String(required=True),
```

Comment:
what's the diff between status_code and trace_state?

Reply:
not sure, just copy this from their example PR.